account_move_csv_import/wizard/account_move_import.py

```diff
@@ -930,7 +930,7 @@ def reconcile_move_lines(self, moves):
                 logger.warning(
                     "Skip reconcile of ref '%s' because the lines with "
                     "this ref have different partners (IDs %s)",
-                    rec_ref, ', '.join(partners.keys()))
+                    rec_ref, ', '.join([str(x) for x in partners.keys() if x not in [False, None]]))
```

Comment:
Hi @njeudy , I'm not sure but what do you think ?
```suggestion
                    rec_ref, ', '.join([str(x) for x in partners.keys() if x]))
```